src/libraries/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.RuntimeOnly.cs

```diff
@@ -401,8 +401,8 @@ public static void Xml_CollectionRoot()
         MyCollection y = SerializeAndDeserialize<MyCollection>(x,
 @"<?xml version=""1.0""?>
 <ArrayOfAnyType xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">
-  <anyType xmlns:q1=""http://microsoft.com/wsdl/types/"" xsi:type=""q1:char"">97</anyType>
-  <anyType xsi:type=""xsd:int"">45</anyType>
+ <anyType xmlns:q1=""http://microsoft.com/wsdl/types/"" xsi:type=""q1:char"">97</anyType>
+ <anyType xsi:type=""xsd:int"">45</anyType>
```

Comment:
What caused the indentation to change? This is something which users have been very sensitive to in the past.

Reply:
@mconnew Is it possible that this is just a visualization problem of git diff?
Initially I tried to extend this test with more primitive types, but received feedback that it is not a good idea to touch old tests. I tried to revert changes and introduced a new test below.

Reply:
You have removed one space from the front of each of these two lines. It's not a visualization issue. It's possible the test doesn't care about the indentation, but I would like to know that before accepting an indentation change in test data. The easiest way is to put the spaces back.

Reply:
@mconnew I've already changed it here(added space): https://github.com/dotnet/runtime/pull/76436/commits/f4f5a9eb411883c1b8fc851b7ac528a1727be392
but git still shows difference.

Reply:
Looks fixed